README.md

```diff
@@ -1,295 +1,202 @@
-# CLI Essentials: Basic Commands in Bash
+# CLI Essentials: Working with Files and Directories in Bash
 
 ## Learning Goals
 
-* Demonstrate how to navigate with Bash
-* Demonstrate manipulating files in the command line
+* Demonstrate interacting with files and directories in the command line
 
 ## Introduction
 
-## Demonstrate How to Navigate With `bash`
+Learning command line tools for managing the file system in Bash can give you
+more speed and control over your workflow. It's actually designed for us to be
```

Comment:
For example, imagine a directory full of files that are named with a year, month, date, time in 24 hour clock timestamp: `file-2018-06-03T1344`. What if we said move all files from 2018 or 2019 that were stamped in the afternoon (the number after `T` is greater than 1200). In the CLI that is one, easy command (it's at the end!). Using a graphical file explorer, this would be a whoooole lot of clicking and thinking.

Reply:
"The CLI is designed to help us be ~lazy~ efficient. So let's let the computer help us!" ;)